test/src/unit-deserialization.cpp

```diff
@@ -358,7 +358,7 @@ TEST_CASE("deserialization")
 
             SECTION("from std::array")
             {
-                std::array<uint8_t, 5> v { {'t', 'r', 'u', 'e'} };
+                std::array<uint8_t, 4> v { {'t', 'r', 'u', 'e'} };
```

Comment:
Could you please add another test case for `std::array<uint8_t, 5> v { {'t', 'r', 'u', 'e', 0} };`?

Reply:
It should be fine if the parser has the `strict=true` option, which is the default. But with `strict=false` this should be OK. So maybe this should be two tests.